src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
+    private int top;
+    private int bottom;
+    private int side;
+
+    public IsoscelesTrapezoid(String color, int top, int bottom, int side) {
+        super(color);
+        this.side = side;
+        this.bottom = bottom;
+        this.top = top;
+    }
+
+    @Override
```

Comment:
The `draw` method is currently responsible for both constructing a message and printing it, which are conceptually different responsibilities. It would be better to separate these concerns, for example, by creating a separate method that returns the string representation of the trapezoid and then printing it in the `draw` method.